components/Dialogs/examples/DialogsThemedPresentationExampleViewController.swift

```diff
@@ -0,0 +1,196 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialDialogs
+import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
+import MaterialComponentsBeta.MaterialDialogs_Theming
+
+class CustomDialogViewController: UIViewController {
+  var dialogTitle: String?
+  var dialogMessage: String?
+
+  init(title: String, message: String) {
+    super.init(nibName: nil, bundle: nil)
+    dialogTitle = title
+    dialogMessage = message
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.backgroundColor = UIColor.white
+
+    let titleLabel = UILabel()
+    titleLabel.text = dialogTitle
+    titleLabel.translatesAutoresizingMaskIntoConstraints = false
+    titleLabel.numberOfLines = 0
+    self.view.addSubview(titleLabel)
+
+    let bodyLabel = UILabel()
+    bodyLabel.text = dialogMessage
+    bodyLabel.translatesAutoresizingMaskIntoConstraints = false
+    bodyLabel.numberOfLines = 0
+    self.view.addSubview(bodyLabel)
+
+    let confirmButton = UIButton()
+    confirmButton.setTitle("OK", for: .normal)
+    confirmButton.setTitleColor(UIColor.black, for: .normal)
```

Comment:
Please use the Buttons Theming Extension to style the button.

Reply:
This one is UIButton intentionally because it is custom view. Do we also prefer to use MDCButton in custom view as a convention?

Reply:
I think we should stick to Material-styled components when we build things for catalogs.  I believe the goal is for our Catalog to beautifully express Material Design on iOS. Even though it's a custom view, it can still be a Material custom view.

Reply:
It makes sense. Addressed in 985551b292491379b5228a975388f1deac15e974.